catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -223,7 +223,8 @@ private QueryRequest getQueryRequestFromIds(Set<String> ids) {
 
   private List<Map<String, Object>> findSubscriptions() {
     try {
-      return persistentStore.get(SubscriptionConstants.SUBSCRIPTIONS_TYPE, "", 0, maxSubscriptions);
+      return persistentStore.get(
+          PersistenceType.SUBSCRIPTION_TYPE.toString(), "", 0, maxSubscriptions);
```

Comment:
Ideally, we shouldn't be using raw persistent store anywhere since we have the subscriptionpersistentstore implementation that we are using to hide the PersistentStore... but since you're just fixing the strings its out of scope. Just something we can look for in the future.